python_cross_site_scripting/QGB_QPSU/HTML.py

```
@@ -0,0 +1,2126 @@
+	assert len(char) == 1
+
+	font = ImageFont.truetype(fm.findfont(fm.FontProperties(family=font_name)), int(size * 0.8))
+	img = Image.new('RGBA' if background == 'transparent' else 'RGB', (size, size), 
+					(0, 0, 0, 0) if background == 'transparent' else ImageColor.getrgb(background))
+	
+	draw = ImageDraw.Draw(img)
+	w, h = draw.textbbox((0, 0), char, font=font)[2:]
+	draw.text(((size - w) // 2, (size - h) // 2 - size // 18), #  18 让上下居中
+			  char, fill=ImageColor.getrgb(foreground), font=font)
+	
+	from qgb import pil;return pil.pil_image_to_bytes(img,'png') if background=='transparent' else pil.pil_image_to_bytes(img,'bmp')
```

Ruff (0.13.3)

Multiple statements on one line (semicolon)

(E702)